contentctl/actions/detection_testing/infrastructures/DetectionTestingInfrastructure.py

```diff
@@ -269,17 +269,25 @@ def configure_imported_roles(
     ):
         indexes.append(self.sync_obj.replay_index)
         indexes_encoded = ";".join(indexes)
+        
         try:
+            # Set which roles should be configured. For Enterprise Security/Integration Testing,
+            # we must add some extra foles.
+            if self.global_config.enable_integration_testing:
+                roles = imported_roles + enterprise_security_roles
+            else:
+                roles = imported_roles
+
             self.get_conn().roles.post(
                 self.infrastructure.splunk_app_username,
-                imported_roles=imported_roles + enterprise_security_roles,
+                imported_roles=roles,
```

Comment:
Fixes the following warning message when testing with Enterprise Security:
<img width="1375" alt="image" src="https://github.com/user-attachments/assets/da8fa39a-2a25-4f7d-a74f-b6ca727ec7cf">